test/e2e/kubectl/kubectl.go

```diff
@@ -115,8 +115,7 @@ var (
 var (
 	proxyRegexp = regexp.MustCompile("Starting to serve on 127.0.0.1:([0-9]+)")
 
-	cronJobGroupVersionResourceAlpha = schema.GroupVersionResource{Group: "batch", Version: "v2alpha1", Resource: "cronjobs"}
-	cronJobGroupVersionResourceBeta  = schema.GroupVersionResource{Group: "batch", Version: "v1beta1", Resource: "cronjobs"}
+	cronJobGroupVersionResourceBeta = schema.GroupVersionResource{Group: "batch", Version: "v1beta1", Resource: "cronjobs"}
```

Comment:
is an e2e test for v1 coming in a follow up PR?

Reply:
+1 for this question. The typical process has been:
* before v1 promotion, we get 100% coverage of the beta API in place (current status visible at https://apisnoop.cncf.io/1.21.0/beta/batch)
* show the beta API tests are stable
* flip the test to use v1 in the PR that introduces the v1 API
* open a follow-up PR promoting the v1 tests to conformance

Reply:
Conformance tests are being promoted in https://github.com/kubernetes/kubernetes/pull/99197, so followup.

Reply:
it's fine to promote to conformance in a follow up, but there needs to be full coverage demonstrated on the v1beta1 API now

Reply:
See https://github.com/kubernetes/kubernetes/pull/99314